app/uk/gov/hmrc/apinotificationpull/util/XmlBuilder.scala

```diff
@@ -19,14 +19,6 @@ package uk.gov.hmrc.apinotificationpull.util
 import uk.gov.hmrc.apinotificationpull.model.Notifications
 
 object XmlBuilder {
```

Comment:
In API-2932, I created a package named `presenters` which contains a `NotificationPresenter` Its job is very similar to this builder. Perhaps this could be called `NotificationsXmlPresenter` and live in the same package?

Reply:
I would use a `util` package rather than `presenter`.
Let's see what other people think